src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -113,6 +116,9 @@ static constexpr hrt_abstime T_ALT_TIMEOUT = 1_s; // time after which we abort l
 static constexpr float THROTTLE_THRESH =
 	0.05f;	///< max throttle from user which will not lead to motors spinning up in altitude controlled modes
 
+static constexpr hrt_abstime T_WIND_EST_TIMEOUT =
+	10_s; // time after which the wind estimate is disabled if no longer updating
```

Comment:
I think it depends on who will actually be wanting to change this often. Also will likely depend on how we improve the wind poll / which estimator we are using. I'm ok to make it a parameter if others think it makes sense now - otherwise we can always add it later if need be.